js/settings.js

```diff
@@ -56,6 +63,11 @@ require.scopes.settings =(() => {
     }
 
     function updateSetting(name, value) {
+        if (!isReady) {
+            console.error('Settings: tried to update before settings is loaded')
```

Comment:
For the sake of uniformity, let's opt for `console.warn` vs `console.error` in cases like this, similar to indexed-db.es6.js
https://github.com/duckduckgo/chrome-zeroclickinfo/blob/beta/js/indexed-db.js#L58